mantidproject/mantid
# Remove UserAlgorithms from packages

Fixes #15206

**This can only be tested on windows**

The `UserAlgorithms` package should no longer be built. The  user algorithms still exist, they can now be found in `Framework/Examples`.
# To Test
- Ensure that all unit test run and pass 
- Ensure that the User algorithms directory (`Framework/UserAlgorithms`) is no longer present in the source.
- Ensure the `Examples` directory is in the Framework directory of the source code and it contains 6 `.cpp` algorithms (plus their respective headers)
- Run CMake ensuring you check the option `ENABLE_CPACK`
- It would helpful if you could time how long cpack takes to run and let me know!
- Build a windows installer with cpack, install mantid, and check that the following are **NOT** in the `MantidInstall` directory:
  - `Includes`
  - `scons-local`
  - `UserAlgorithms`

**[Release Notes](http://www.mantidproject.org/ReleaseNotes_3_7_Framework_Changes#Deprecated)**
